Code/Framework/AzCore/Tests/Math/SfmtTests.cpp

```diff
@@ -27,8 +27,8 @@ namespace UnitTest
         void SetUp() override
         {
             AllocatorsFixture::SetUp();
-            array1 = (AZ::u64*)azmalloc(sizeof(AZ::u64) * 2 * (BLOCK_SIZE / 4), AZStd::alignment_of<AZ::u64>::value);
-            array2 = (AZ::u64*)azmalloc(sizeof(AZ::u64) * 2 * (10000 / 4), AZStd::alignment_of<AZ::u64>::value);
+            array1 = (AZ::u64*)azmalloc(sizeof(AZ::u64) * 2 * (BLOCK_SIZE / 4), AZStd::alignment_of<AZ::Vector4>::value);
+            array2 = (AZ::u64*)azmalloc(sizeof(AZ::u64) * 2 * (10000 / 4), AZStd::alignment_of<AZ::Vector4>::value);
```

Comment:
this was producing a lot of alignment errors when operating over that memory with SIMD